packages/devextreme-scss/scss/widgets/base/chat/_index.scss

```diff
@@ -0,0 +1,166 @@
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+  margin: 0;
+  overflow-y: auto;
+}
+
+.dx-chat-message-group {
+  display: grid;
+  align-items: start;
+  row-gap: 4px;
+  margin: 24px 0;
+}
+
+.dx-chat-message-group-alignment-start {
+  justify-items: start;
+  grid-template-columns: 44px 1fr;
+}
+
+.dx-chat-message-group-alignment-end {
+  justify-items: end;
+}
+
+.dx-chat-message-group-information {
+  display: flex;
+  grid-row: 1;
+}
+
+.dx-chat-message-group-alignment-start .dx-chat-message-group-information {
+  grid-column: 2;
+}
+
+.dx-chat-message-time,
+.dx-chat-message-name {
+  font-size: $chat-information-font-size;
+  color: $chat-information-color;
+}
+
+.dx-chat-message-name {
+  margin-right: 8px;
+}
+
+.dx-chat-message-avatar {
+  grid-row: span 3;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: $chat-avatar-size;
+  height: $chat-avatar-size;
+  border-radius: 50%;
+  background-color: #878787;
+}
+
+.dx-chat-message-bubble {
+  padding: 8px 12px;
+  max-width: 90%;
+  border-radius: $chat-bubble-border-radius;
+  background-color: $chat-bubble-background-color-socondary;
+}
+
+.dx-chat-message-group-alignment-start .dx-chat-message-bubble.dx-chat-message-bubble-first {
+  border-bottom-left-radius: 0;
+}
+
+.dx-chat-message-group-alignment-start .dx-chat-message-bubble.dx-chat-message-bubble-last {
+  border-top-left-radius: 0;
+}
+
+.dx-chat-message-group-alignment-start .dx-chat-message-bubble-first.dx-chat-message-bubble-last {
+  border-bottom-left-radius: $chat-bubble-border-radius;
+  border-top-left-radius: $chat-bubble-border-radius;
+}
+
+.dx-chat-message-group-alignment-start .dx-chat-message-bubble:not(
+  .dx-chat-message-bubble-first.dx-chat-message-bubble-last,
+  .dx-chat-message-bubble-first,
+  .dx-chat-message-bubble-last) {
+  border-bottom-left-radius: 0;
+  border-top-left-radius: 0;
+}
+
+.dx-chat-message-group-alignment-end .dx-chat-message-bubble.dx-chat-message-bubble-first {
+  border-bottom-right-radius: 0;
+}
+
+.dx-chat-message-group-alignment-end .dx-chat-message-bubble.dx-chat-message-bubble-last {
+  border-top-right-radius: 0;
+}
+
+.dx-chat-message-group-alignment-end .dx-chat-message-bubble-first.dx-chat-message-bubble-last {
+  border-bottom-right-radius: $chat-bubble-border-radius;
+  border-top-right-radius: $chat-bubble-border-radius;
+}
+
+.dx-chat-message-group-alignment-end .dx-chat-message-bubble:not(
+  .dx-chat-message-bubble-first.dx-chat-message-bubble-last,
+  .dx-chat-message-bubble-first,
+  .dx-chat-message-bubble-last) {
+  border-bottom-right-radius: 0;
+  border-top-right-radius: 0;
+}
+
+.dx-chat-message-group-alignment-start .dx-chat-message-bubble {
+  grid-column: 2;
+}
+
+.dx-chat-message-group-alignment-end .dx-chat-message-bubble {
+  background-color: $chat-bubble-background-color-primary;
+}
+
+.dx-chat-message-box {
+  display: flex;
+  align-items: center;
+}
+
+.dx-chat .dx-chat-message-box-text-area {
```

Comment:
We need to come to agreement whether we write `textarea` or `text-area`. I would suggest the latter (you need to change other names with `textarea` though)

Reply:
[87645b61de0f97694a8ad5c82a0e2b8a5937bf99]